sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -3884,14 +3884,12 @@ object TimeWindowing extends Rule[LogicalPlan] {
           case _ => Metadata.empty
         }
 
+
```

Comment:
nit. Please remove this redundant new line addition.

Reply:
> nit. Please remove this redundant new line addition.
@dongjoon-hyun 
ok,thanks!